assets/css/styleguide.scss

```diff
@@ -858,3 +883,126 @@ code[class*="language-"], pre[class*="language-"] {
 .usa-code-sample pre[class*="language-"] {
   margin-top: 0;
 }
+
+.preview {
+  .usa-background-dark {
+    display: inline-block;
+    padding: {
+      left: 1em;
+      right: 1em;
+    }
+  }
+}
+
+.usa-link-visited {
+  color: $color-visited;
+}
+
+.alignment-example {
+  border-left: 2px solid $color-primary-darker;
+  padding-left: 1rem;
+}
+
+.help-text {
+  border-top: 2px solid $color-primary-darker;
+  color: $color-primary-darker;
+  font-size: $em-base;
+  padding-top: 1rem;
+  text-align: center;
+}
+
+.typography-sans-intro {
+  .text-huge {
+    font-size: 140px;
+    line-height: 1;
+  }
+  .text-tiny {
+    font-size: 15px;
+  }
+}
+
+.typography-serif-intro {
+  .text-huge {
+    font-size: 120px;
+    line-height: 1;
+  }
+  .text-tiny {
+    font-size: 13px;
+  }
+}
+
+// Alt font styles
+
+$font-light: 300;
+
+.serif-robust {
+  .usa-display-alt {
+    font-size: $h1-font-size;
+    font-weight: $font-light;
+  }
+
+  .usa-font-lead {
+    font-family: $font-serif;
+    font-size: $h3-font-size;
+  }
+
+  .usa-font-lead-alt {
+    font-size: $h5-font-size;
+  }
+}
+
+.sans-style {
```

Comment:
Is this a pattern library style? Why are these sizes being modified? Shouldn't we be showing the users the exact font sizes they should be getting when they use the library?